mentoring/dataexport.py

```diff
@@ -30,18 +30,30 @@
 from xblock.core import XBlock
 from xblock.fields import String, Scope
 from xblock.fragment import Fragment
+from xblockutils.resources import ResourceLoader
 
-from .models import Answer
-from .utils import list2csv, loader
 
 
 # Globals ###########################################################
 
 log = logging.getLogger(__name__)
 
+# Utils ###########################################################
+
+def list2csv(row):
+    """
+    Convert a list to a CSV string (single row)
+    """
+    f = StringIO()
+    writer = unicodecsv.writer(f, encoding='utf-8')
+    writer.writerow(row)
+    f.seek(0)
+    return f.read()
```

Comment:
@bradenmacdonald you haven't closed `StringIO`. I'm not sure if it's smart enough to close itself when it's garbage-collected, but it would be better to do sometihng like

    with StringIO() as f:
        writer = unicodecsv.writer(f, encoding='utf-8')
        writer.writerow(row)
        return f.getvalue()